notebooks/official/prediction/get_started_with_vertex_private_endpoints.ipynb

```diff
@@ -1058,7 +1038,7 @@
       "source": [
         "### Make the prediction request using SDK\n",
         "\n",
-        "Next, use the `Vertex AI SDK` to make a prediction request."
+        "Next, use the `Vertex AI SDK` to make a prediction request. Request is exeucte if notebook is in VPC network. Hence, prediction request is not executable in google colab as it is not part of VPC netwok."
```

Comment:
- remove back ticks from Vertex AI SDK, and change to Vertex AI SDK for Python.
- suggested rewrite (note typo "exeucte" and other corrections):
-     The request is executed if the notebook is in the VPC network. Therefore, the prediction request isn't executable in google colab since it's not part of the VPC netwok."

Reply:
Did the required changes